ui/main.qml

```diff
@@ -490,33 +501,45 @@ StatusWindow {
         }
     }
 
-    MacTrafficLights { // FIXME should be a direct part of StatusAppNavBar
+    // TODO: This area for MacOs and generic topbar
+    // for with notification and Profile button should probably
+    // be combined here
+    Item {
+        visible: d.showCustomMacOsToolbar
+
         anchors.left: parent.left
+        anchors.right: parent.right
         anchors.top: parent.top
-        anchors.margins: 13
-
-        visible: Qt.platform.os === Constants.mac && applicationWindow.visibility !== Window.FullScreen
 
-        onClose: {
-            if (loader.sourceComponent != app) {
-                Qt.quit()
-                return
-            }
-
-            if (localAccountSensitiveSettings.quitOnClose) {
-                Qt.quit();
-                return
-            }
+        implicitHeight: 40
 
-            applicationWindow.visible = false;
+        MouseArea {
+            anchors.fill: parent
+            preventStealing: true
+            propagateComposedEvents: true
+            onPressed: applicationWindow.startSystemMove()
         }
 
-        onMinimised: {
-            applicationWindow.toggleMinimize()
-        }
+        MacTrafficLights {
```

Comment:
I think now is a good time to clean this up a bit; we have 2 components here `MacTrafficLights` and `StatusMacTrafficLights`, pls use the one from StatusQ and remove the former